### ISSUE TITLE
bug: duplicates of authenticated users in etherpad when resuming to pad

### ISSUE THREAD

**Issue by tommyhlw** [ether/ep_openid_connect#7]
like mentioned in #6 a replications of the authenticated user appear in the pad when resuming to the pad. Can be tested with multiple browser  in parallel or closing browser and authenticate again.

**Comment by rhansen** [comment]
I would love to fix this, but unfortunately it will take a lot of work. Etherpad core and other plugins use the author ID for message routing (which is why Etherpad doesn't allow you to open the same pad in two tabs), which means we would have to audit plugins, figure out a migration path for the affected plugins, and redo the message routing in Etherpad core.

**Comment by rhansen** [comment]
There is a halfway solution: When a user logs in, always give them the same author ID. If the user tries to visit the same pad twice (whether in different tabs of the same browser or in different browsers), disconnect the older login. This would be much easier to implement.

**Comment by rhansen** [comment]
This is addressed by the new [ep_stable_authorid](https://github.com/ether/ep_stable_authorid#readme) plugin. That plugin requires running current Etherpad `develop` branch because it relies on the [new `getAuthorId` hook](https://github.com/ether/etherpad-lite/pull/5460) that will be released when v1.9.0 is ready.
